app/components/CurrencySelect.tsx

```diff
@@ -0,0 +1,169 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+const GradientButtonWrapper = styled.div<{ isOpen: boolean }>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 60px;
+  background: linear-gradient(
+    90deg,
+    #170557 0%,
+    #4489ce 0%,
+    #944ba9 100%,
+    #00d1ff 100%
+  );
+  border-radius: ${({ isOpen }) => (isOpen ? "16px 16px 0 0" : "16px")};
+`;
+
+const GradientButton = styled.button<{ isOpen: boolean }>`
+  width: calc(100% - 8px);
+  height: calc(100% - 8px);
+  cursor: not-allowed;
+  background-color: #060134;
+  color: #ffffff;
+  border: none;
+  border-radius: ${({ isOpen }) => (isOpen ? "12px 12px 0 0" : "12px")};
+  font-size: 17px;
+  text-transform: uppercase;
+  font-family: "GT America CM";
+  letter-spacing: 0.46px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 8px 16px;
+
+  .sc {
+    color: #00d0fe;
+    text-transform: uppercase;
+    font-size: 17px;
+    font-family: "GT America Condensed Bold";
+    letter-spacing: 0.41px;
+    margin-top: 0;
+    margin-bottom: 4px;
+  }
+
+  p {
+    font-size: 17px;
+    letter-spacing: 0.46px;
+    color: #ffffff;
+    text-transform: uppercase;
+    margin: 0;
+  }
+`;
+
+const currencies = [
+  {
+    key: "sUSD",
+    name: "Synthetic US Dollars",
+  },
+  {
+    key: "sSNX",
+    name: "Synthetix Token",
+  },
+  {
+    key: "sEuro",
+    name: "Synthetic Euro",
+  },
+];
+
+// Note: The currency select feature is currently disabled (see line 85).
+// This is because this application only supports swapping ETH for now.
+
+const CurrencySelect = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <>
+      <GradientButtonWrapper isOpen={isOpen}>
+        <GradientButton onClick={handleToggle} isOpen={isOpen} disabled>
+          {/* eslint-disable-next-line */}
+          <DropdownImage
+            isOpen={isOpen}
+            src="/img/control-panel-arrow.svg"
+            alt="Conrol Panel Arrow"
+          />
+          <div>
+            <p className="sc">Select currency</p>
+            <p>Ethereum (ETH)</p>
+          </div>
+          {/* eslint-disable-next-line */}
```

Comment:
What is eslint complaining about? Let's try not to use eslint-disable if possible